nusslein_vsrp/latex/overview.tex

```diff
@@ -655,59 +760,50 @@ \subsection{Choice of baseline method}
-If the classical RK4 method is used the order has to be reduced more because the RK4 method does not have embedded methods of order~3.
-In contrast to this, some methods with $s > p$ have degrees of freedom even without reducing the order.
+For explicit methods with the number of stages equal to the order of the
+method, the order must be reduced in order to allow any freedom in the weights.
+If the classical RK4 method is used the order has to be reduced more because
+the RK4 method does not have embedded methods of order~3.
+In contrast to this, some methods with $s > p$ admit changes to the weights without reducing the order.
 An example of this is SSPRK(10,4), that has 4 degrees of freedom for $\pt = p$.
-For Cash--Karp RK5 and Dormand--Prince RK5 the order has to be reduced to get degrees of freedom even though the number of stages is higher than the order.
+For Cash--Karp RK5 and Dormand--Prince RK5, even though the number of stages is higher than the order,
+the order must be reduced in order to allow any modification of the weights.
 
-
-In Table\,\ref{table:DOF_imp}, the same properties of some implicit
-methods are reported.
-We can see that the fully implicit methods Lobatto~IIIC4 and Radau~IIA3 require a drastic reduction of the order as expected.
+Regarding implicit methods,
+we can see that the fully implicit methods Lobatto~IIIC4 and Radau~IIA3 require a drastic reduction of the order, as expected.
 The diagonally implicit SDIRK(5,4) method only requires an order reduction of one to get one degree of freedom for the weights.
 The TR-BDF2 method even allows adaptations without reducing the order.
 The implicit Euler extrapolation methods also exhibit degrees of freedom without a reduction of the order.
 
-Another important property is the stability region.
-The adaption of the weights does not solve issues with stability when used with a too big $\dt$. Therefore, the RK method has to be stable for large $\dt$.
+It is also desirable that the baseline method have a large stability region.
 
-A third property is the existence of a positive solution. This corresponds to the question whether there is an embedded first order method that ensures positivity. This is particularly interesting for implicit methods, because $\dt$ can be unrestricted by stability requirements.
-If we have an embedded first order method, we can ensure that there is always a positive solution, which might be of first order.
-This is the case for the Im-Euler extrapolation methods, because they contain a chain of BE steps.
-For explicit methods the FE method is always embedded, but a reduction of the step size might be required.
+Note that for any diagonally implicit method, by allowing the order to be reduced to one
+we can guarantee the existence of a solution to the LP, since the backward Euler method
+is unconditionally positive.  For explicit methods, reducing the order to one is guaranteed
+to yield a solution of the LP only if the step size is small enough.
```

Comment:
I think this is not necessarily true. For example, the implicit midpoint method (Gauß method of order 2) does not contain an embedded implicit Euler method but is diagonally implicit.

Reply:
You're right.  I've rephrased this to be correct.